modules/core/src/v2/coins/algo.ts

```diff
@@ -443,3 +443,62 @@ export class Algo extends BaseCoin {
     return Bluebird.resolve(true).asCallback(callback);
   }
 }
+
+// -------------------------------------------------------------------------------------
+// Module Augmentations
+// -------------------------------------------------------------------------------------
+
+// Augments the algosdk module to include types from the BitGo algosdk fork.
+//
+// This is required because latest version of algosdk that is installed in
+// account-lib cause compilation to fail here without augmentation.
+//
+// TODO: Remove augmentation after algosdk fork is completely replaced by
+// account lib.
+declare module 'algosdk' {
```

Comment:
Module augmentations should really go in https://github.com/BitGo/BitGoJS/tree/master/types so we can use them across all packages in the monorepo

Reply:
I don't think we would want this because in account-lib where we have the latest version of the SDK installed, these augmentations don't exist because they only exist in the fork.

Reply:
Fair enough, this is fine with me as it is then